src/os/bluestore/BlueFS.cc

```diff
@@ -3062,8 +3062,8 @@ int64_t BlueFS::_maybe_extend_log() {
   // increase with respect the number of extents. bluefs_min_log_runway should cover the max size 
   // a log can get.
   // inject new allocation in case log is too big
-  if (log.t.expected_size() > runway) {
```

Comment:
To be super correct we should have some calculation that:
assume we allocate bluefs_max_log_runway by 4K sized extents;
the size of extents should be smaller then super.block_size.

Note: I have seen a deployment when super.block_size was 512.

Reply:
As per our discussion offline. We went for having margin of min_log_runway as it is less complex and it should be big enough so that it doesn't cause problems